lib/src/sai_redis_generic_get_stats.cpp

```diff
@@ -0,0 +1,192 @@
+#include "sai_redis.h"
+#include "meta/saiserialize.h"
+
+sai_status_t internal_redis_get_stats_process(
+        _In_ sai_object_type_t object_type,
+        _In_ uint32_t count,
+        _Out_ uint64_t *counter_list,
+        _In_ swss::KeyOpFieldsValuesTuple &kco)
+{
+    SWSS_LOG_ENTER();
+
+    // key:         sai_status
+    // field:       stat_id
+    // value:       stat_value
+
+    const std::string &key = kfvKey(kco);
+    const std::vector<swss::FieldValueTuple> &values = kfvFieldsValues(kco);
```

Comment:
>const std::vector<swss::FieldValueTuple> [](start = 4, length = 40)

suggest use 'auto' to make code readable.

Reply:
Done